build/changelog/changelog.go

```diff
@@ -127,15 +130,26 @@ func main() {
 		fail(err)
 	}
 
-	var rgx *regexp.Regexp
+	var rgx, rgxAlt, rgxDeps *regexp.Regexp
 	if len(plugin) > 0 {
 		// craft a regex to filter all plugin-related commits that follow
 		// the conventional commit format
 		rgx, _ = regexp.Compile("^[a-f0-9]+ [a-zA-Z]+\\(([a-zA-Z\\/]+\\/)?" + plugin + "(\\/[a-zA-Z\\/]+)?\\):.*")
+
+		// alternative plugin name used as scope
+		if plugin == "k8saudit" {
+			pluginAlt := "k8s_audit"
+			rgxAlt, _ = regexp.Compile("^[a-f0-9]+ [a-zA-Z]+\\(([a-zA-Z\\/]+\\/)?" + pluginAlt + "(\\/[a-zA-Z\\/]+)?\\):.*")
+		}
```

Comment:
Why do we need this?

Is there any valid alternative that does not require hard code the plugin name? :thinking:

Reply:
There are a few commits for release 0.8 that use a different conventional commit scope (`k8s_audit` instead of `k8saudit`, e.g.:

```
v0.8
cleanup(plugins/k8s_audit): make the rulesefile self-referenced

v0.6
feat(plugins/k8saudit): include query params in health check endpoint exceptions
```

Pretty much summed up by https://github.com/falcosecurity/plugins/issues/148

Reply:
A possible solution for the general case: 
since [k8s_audit](https://github.com/falcosecurity/plugins/blob/0e7ef76987476224a799e6db8af57b5716040b30/registry.yaml#L58) is the _source_ name (which may differ from the _plugin_ name), we may use the _source_ as an alternative name used as scope.

If we want to postpone this fix for a follow-up PR, I'm okay with that. In such a case, please leave a comment in the code (e.g., `fixme: ....`) explaining why we needed that workaround.

Reply:
checking the commit scope if it contains the event source name now